Hazel/src/Platform/OpenGL/OpenGLRendererAPI.cpp

```diff
@@ -4,9 +4,33 @@
 #include <glad/glad.h>
 
 namespace Hazel {
+#ifdef HZ_ENABLE_RENDERER_LOG
+	void OpenGLRendererAPI::OpenGLMessageCallback(
+		unsigned source,
+		unsigned type,
+		unsigned id,
+		unsigned severity,
+		int length,
+		const char* message,
+		const void* userParam)
+	{
+		switch (severity)
+		{
+			case GL_DEBUG_SEVERITY_HIGH:			HZ_CORE_CRITICAL(message); return;
+			case GL_DEBUG_SEVERITY_MEDIUM:			HZ_CORE_ERROR(message); return;
+			case GL_DEBUG_SEVERITY_LOW:				HZ_CORE_WARN(message); return;
+			case GL_DEBUG_SEVERITY_NOTIFICATION:	HZ_CORE_TRACE(message); return;
```

Comment:
After case and the HZ_CORE, indent with spaces instead of tabs (like done in log.h where the macro's are defined), ad different tab lenghts result in different indentations required (visible on github versus VS)